pandas/tests/frame/methods/test_cov_corr.py

```diff
@@ -58,6 +58,16 @@ def test_cov(self, float_frame, float_string_frame):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("test_ddof", [0, 1, 2, 3])
+    def test_cov_ddof(self, test_ddof):
+        np_array1 = np.random.rand(10)
+        np_array2 = np.random.rand(10)
+        df = DataFrame({0: np_array1, 1: np_array2})
+        result = df.cov(ddof=test_ddof)
```

Comment:
Can you also have a test with the the value unspecified so that the default is used?

Reply:
Done